KubeArmor/core/kubeUpdate.go

```diff
@@ -2771,3 +2787,47 @@ func (dm *KubeArmorDaemon) GetConfigMapNS() string {
 	}
 	return envNamespace
 }
+
+// isVersionGreaterThanOrEqual checks if the Kubernetes version  >= specified version.
+func isVersionGreaterThanOrEqual(version, requiredVersion string) bool {
```

Comment:
1. Is it possible to use semver package instead of implementing this ourselves? Ref - https://pkg.go.dev/golang.org/x/mod/semver
2. Can we make sure this logic works with Kubernetes distributions like k3s? The version they give has some additional patch strings added.